ansible-ipi-install/roles/installer/tasks/main.yml

```diff
@@ -3,6 +3,7 @@
 - include_tasks: 10_get_oc.yml 
 - include_tasks: 20_extract_installer.yml
 - include_tasks: 30_create_metal3.yml
+  when: (release_version[0]|int >= 4) and (release_version[2]|int <= 3)
```

Comment:
same as above comment.

Reply:
Yeah, I was just thinking about the conditional while grabbing lunch.  Needs a tweak. Will fix!

Reply:
Just FYI, same can be said about the #153 where the same logic was applied for install-config.j2 which we merged in already.

Reply:
Should I just make that change on this PR as well?

Reply:
no, after we merge this. lets just make a separate PR for the #153